websockify/websocketserver.py

```diff
@@ -35,6 +36,13 @@ class WebSocketRequestHandler(BaseHTTPRequestHandler):
     def __init__(self, request, client_address, server):
         BaseHTTPRequestHandler.__init__(self, request, client_address, server)
 
+    def address_string(self, show_proxied=False):
+        immediate_client = super(WebSocketRequestHandler, self).address_string()
+        fwd_for = self.headers.get("X-Forwarded-For", None)
+        if show_proxied and fwd_for:
+            return "{},{}".format(fwd_for, immediate_client)
```

Comment:
This format string isn't compatible with Python 2.6, which is our current baseline.